latex/epoch.tex

```diff
@@ -146,11 +269,11 @@ \subsection{Stake Distribution Calculation}
       \nextdef
       & \fun{rewardStake} \in (\AddrRWD\mapsto\Coin) \to \Stake \\
       & \fun{rewardStake}~{rewards} =
-          \{\fun{stakeHK_r}~{addr}\mapsto c \mid a\mapsto c\in\var{rewards}\} \\
+          \{\fun{stakeHK_r}~{a}\mapsto c \mid a\mapsto c\in\var{rewards}\} \\
```

Comment:
Are we filtering the stake that belongs to staking keys that are not registered anywhere, so that unregistered staking keys do not add to the size of the local state?

Reply:
Ah, we do, in the next figure! :)